src/libparms/db/parmsdbappstate.h

```diff
@@ -35,4 +35,5 @@
 #define UPDATE_APP_STATE_REQUEST_ID "update_value_with_key"
 #define UPDATE_APP_STATE_REQUEST "UPDATE app_state SET value=?2 WHERE key=?1;"
 
-#define APP_STATE_KEY_DEFAULT_Test "Test"
\ No newline at end of file
+#define APP_STATE_KEY_DEFAULT_LastServerSelfRestart "0"
+#define APP_STATE_KEY_DEFAULT_LastClientSelfRestart "0"
```

Comment:
As described in the SonarCloud issue, prefer using const value instead of macro (the remaining of the code will be refactored later, but let's apply it now on new code)